.github/workflows/pull_request.yml

```diff
@@ -1,38 +1,25 @@
-name: Build Chain
+name: Build Chain [pull request]
 
 on: [pull_request]
 
 jobs:
-  build-chain-openjdk8:
+  build-chain:
+    strategy:
+      matrix:
+        java-version: [8, 11]
     runs-on: ubuntu-latest
-    name: Pull Request openjdk8
+    name: Build Pull Request
```

Comment:
Isn't there duplicated phrase  "Pull Request" too many times? I would remove some to keep it once only in it
Example  `Build Chain [pull request] / Build Pull Request (8) (pull_request) `

Reply:
changed to `Build Chain / Maven Build (${{ matrix.java-version }}) (%EVENT_TYPE%) `